src/main/java/net/sf/jabref/gui/desktop/JabRefDesktop.java

```diff
@@ -265,6 +280,14 @@ public static void openBrowser(String url) throws IOException {
         openExternalFilePlatformIndependent(fileType, url);
     }
 
+    public static void openBrowserCatchException(String url) {
+        try {
+            openBrowser(url);
+        } catch (IOException exception) {
+            LOGGER.error("Error while opening following ling with a browser: "+ url, exception);
```

Comment:
Shouldn't there already exist similar methods and translation strings
somewhere in JabRef? JabRef can open links since ages!